packages/@aws-cdk/core/README.md

```diff
@@ -465,6 +465,7 @@ export class Sum extends Construct {
     const serviceToken = CustomResourceProvider.getOrCreate(this, resourceType, {
       codeDirectory: `${__dirname}/sum-handler`,
       runtime: CustomResourceProviderRuntime.NODEJS_12,
+      environment: { STAGE: 'beta' },
```

Comment:
This is not relevant to the current example, so please don't do this just to satisfy the PR linter.

It's telling you to add a section, describing the feature and why you might use it (in addition to adding an example).

Reply:
Got it. Do you think this particular change needs a different section or a change to README at all?

Reply:
I'm okay with it not having a README section.

Reply:
Alright, will remove.